install/homebox-install.sh

```diff
@@ -37,7 +41,29 @@ After=network.target
 
 [Service]
 WorkingDirectory=/opt
-ExecStart=/opt/homebox
+ExecStart=/opt/homebox \\
```

Comment:
All Not needed, 

```bash
[Service]
WorkingDirectory=/opt
ExecStart=/opt/homebox
EnvironmentFile=/opt/.env
Restart=on-failure
```
should be enough

Reply:
Did you try just using just the `.env` file with environment variables as MickLesk suggested? To me, this documentation reads like you can _choose_ between using either cli args `--mode` **or** env variable `$HBOX_MODE`. And it would be preferable to stick with only env variables.

```
OPTIONS
--mode/$HBOX_MODE                                                        <string>  (default: development)
--web-port/$HBOX_WEB_PORT                                                <string>  (default: 7745)
--web-host/$HBOX_WEB_HOST                                                <string>
--web-max-upload-size/$HBOX_WEB_MAX_UPLOAD_SIZE                          <int>     (default: 10)
--storage-data/$HBOX_STORAGE_DATA                                        <string>  (default: ./.data)
--storage-sqlite-url/$HBOX_STORAGE_SQLITE_URL                            <string>  (default: ./.data/homebox.db?_fk=1)
--log-level/$HBOX_LOG_LEVEL                                              <string>  (default: info)
--log-format/$HBOX_LOG_FORMAT                                            <string>  (default: text)
--mailer-host/$HBOX_MAILER_HOST                                          <string>
--mailer-port/$HBOX_MAILER_PORT                                          <int>
--mailer-username/$HBOX_MAILER_USERNAME                                  <string>
--mailer-password/$HBOX_MAILER_PASSWORD                                  <string>
--mailer-from/$HBOX_MAILER_FROM                                          <string>
--swagger-host/$HBOX_SWAGGER_HOST                                        <string>  (default: localhost:7745)
--swagger-scheme/$HBOX_SWAGGER_SCHEME                                    <string>  (default: http)
--demo/$HBOX_DEMO                                                        <bool>
--debug-enabled/$HBOX_DEBUG_ENABLED                                      <bool>    (default: false)
--debug-port/$HBOX_DEBUG_PORT                                            <string>  (default: 4000)
--options-allow-registration/$HBOX_OPTIONS_ALLOW_REGISTRATION            <bool>    (default: true)
--options-auto-increment-asset-id/$HBOX_OPTIONS_AUTO_INCREMENT_ASSET_ID  <bool>    (default: true)
--options-currency-config/$HBOX_OPTIONS_CURRENCY_CONFIG                  <string>
--help/-h    display this help message
```

Reply:
as I said in my reply it will not work. I have tested it multiple times it's either including the cli Args or you have to use the option in the env file

Reply:
I just tested and using just `.env` file works fine, please use that instead 
![1](https://github.com/user-attachments/assets/43baa557-cfa2-4d29-b121-36d15a708a02)
![2](https://github.com/user-attachments/assets/68e08858-0b38-4c23-8a51-72da8af726a3)
![3](https://github.com/user-attachments/assets/ad6fe901-e45c-4b7d-9c1a-f634adf74dce)

Reply:
strange in my testing they just don't work

Reply:
How does your `.env` file look? You cannot have comments after the environment variables, then it will crash. Check service logs `journalctl -u homebox.service -n 20`

Reply:
like yours no comments at all homebox runs fine just not on the set port also tried restarting the LXC

Reply:
What if you just add `HBOX_LOG_FORMAT=json`, does your logs change to json?

![image](https://github.com/user-attachments/assets/c16d9b8e-1118-46fe-a83d-3f9676e114bb)

Reply:
Assuming you are restarting the service (`systemctl restart homebox.service`) after changing `.env`? Also doublecheck your systemd file is correct. Not sure what's the problem on your end, but works fine here

Reply:
I just did a fresh install with a new CT and it's working fine no clue what was going on with my other one